### ISSUE TITLE
Missing RaptorLake PCI Device ID

### ISSUE THREAD

**Issue by fcclbastianboese** [chipsec/chipsec#1636]
Running chipsec on a mainboard with RaptorLake pch and cpu

```
python368.efi chipsec_main.py -m common.spi_access
ERROR: Unknown Platform: VID = 0x8086, DID = 0xA707, RID = 0x00
```
See for reference:
https://github.com/torvalds/linux/blob/master/arch/x86/events/intel/uncore_snb.c

**Comment by npmitche** [comment]
We will work on it! 

TODO: Add RPL.xml from datasheet 743844 and 743846

**Comment by npmitche** [comment]
Fixed in https://github.com/chipsec/chipsec/pull/1653 and released in [tag 1.10.1 ](https://github.com/chipsec/chipsec/releases/tag/1.10.1)
